src/Microsoft.TestPlatform.ManagedNames/ManagedNameUtilities/ManagedNameParser.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.ManagedNameUtilities
 
     public class ManagedNameParser
```

Comment:
Maybe this class should be internal too ?

Reply:
We like to provide these helper methods mainly for adapter authors. That's the reason they are public.